geocatbridge/servers/views/geonetwork.ui

```diff
@@ -115,6 +115,57 @@
           </property>
          </widget>
         </item>
+        <item row="4" column="0">
+         <widget class="QLabel" name="label_33">
+          <property name="text">
+           <string>Group</string>
+          </property>
+         </widget>
+        </item>
+        <item row="4" column="1">
+         <widget class="QLineEdit" name="txtGeonetworkGroup">
+          <property name="text">
+           <string/>
+          </property>
+          <property name="placeholderText">
+           <string>Group Id</string>
+          </property>
+         </widget>
+        </item>
+        <item row="5" column="0">
+         <widget class="QLabel" name="label_35">
+          <property name="text">
+           <string>Assign to current Catalogue</string>
```

Comment:
This is rather long imho. Need to clone your PR and see if the UI still looks okay...
Either way, I'd prefer a lowercase "c" for "catalogue". Also not sure about the British English spelling (consistency)...

Reply:
Also, since this is a boolean, we should perhaps turn this into a radio button or something 🤔

Reply:
British spelling seems to dominate but not always.